docker/Dockerfile

```diff
@@ -11,5 +11,9 @@ RUN mkdir -p $GOPATH/src/github.com/algorand
 WORKDIR $GOPATH/src/github.com/algorand
 RUN git clone https://github.com/algorand/go-algorand
 WORKDIR $GOPATH/src/github.com/algorand/go-algorand
+RUN apt-get -y install sudo
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get -y install tzdata
+RUN apt-get -y install bsdmainutils
```

Comment:
What requires bsdmainutils and tzdata?  I realize sudo is now assumed to exist but it's not clear why these 2 are required.

Reply:
Hexdump is in bsdmainutils, which is used by some of the configure/build scripts

Reply:
For me, at least, the build fails on the call to scripts/configure_dev.sh , when apt is updated and the system attempts to interactively configure tzdata by prompting the user to enter their locale. Probably there are better-ways to avoid this happening.

Reply:
Thanks - didn't realize we had hexdump dependency.  The tzdata interaction is something i've seen and fixed like this.  Do we have a dependency on tzdata?

Reply:
Not quite a dependency, but when apt update is called by various utils in the repo, tzdata will be attempted to install interactively, prompting the user for input, halting docker build. I install tzdata silently to get around it.